lib/eslint/flat-eslint.js

```diff
@@ -326,14 +326,15 @@ async function loadFlatConfigFile(filePath) {
  * as override config file is not explicitly set to `false`, it will search
  * upwards from the cwd for a file named `eslint.config.js`.
  * @param {import("./eslint").ESLintOptions} options The ESLint instance options.
- * @returns {{configFilePath:string,basePath:string}} Location information for
+ * @returns {{configFilePath:string,basePath:string,error:boolean}} Location information for
  *      the config file.
  */
 async function locateConfigFileToUse({ configFile, cwd }) {
 
     // determine where to load config file from
     let configFilePath;
     let basePath = cwd;
+    let error = false;
```

Comment:
The name `error` sounds generic and I expected it to be an instance of `Error`. Can we use something like `isConfigFileDetected`/`hasConfigFile`? I feel it's better suited for a boolean value.

Reply:
Or we can return an error object `error = new Erro("Could not find config file.");`, which can be thrown directly where it is used?

Reply:
sure, will do!